src/main/c/Makefile

```diff
@@ -239,6 +239,13 @@ freebsd64: export platform := freebsd/x86_64
 freebsd64:
 	$(MAKE) -f natives.mk
 
+# Requires a FreeBSD host running on aarch64 (arm64) architecture.
+freebsdarm64v8: export CFLAGS += -I$(JAVA_HOME)/include/freebsd -m64
+freebsdarm64v8: export LDFLAGS += -m64
```

Comment:
I think [the `-m64` flag](https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html#index-m64-5) is AMD64-specific, and I suspect it's ignored outright when building for AArch64. Have you tried without the flag to confirm it's necessary, or if not, can you please do so?

Reply:
Your right! I missed that one. I have removed the flag and verified the resulting binary still works.